src/Controls/src/Core/Page/Page.Mapper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+namespace Microsoft.Maui.Controls;
+
+public partial class Page
+{
+	public static IPropertyMapper<IContentView, IPageHandler> ControlsPageMapper =
```

Comment:
@mattleibow should this use `ModifyMapping` instead of adding a new `ControlsPageMapper` ?

Reply:
Yea, I wonder if we should eventually just add builder extension methods for modifying mappers. In my own code I typically always do it inside `ConfigureHandlers`, but we don't really hold users hands to that pit of success. 

Maybe we can throw a warning from the helpers if they are called too early?

For the immediate needs of this PR. `ReplaceMapping` probably sounds good. 

If I'm following you then we'd just add `ReplaceMapping` here?

https://github.com/dotnet/maui/pull/15512/files#diff-e7da121e9db2473eac3f106be1b46f8fc5d94eeef4bb730d5ea84a16d2c32a4fR17

```C#
	internal static new void RemapForControls()
	{
		// Adjust the mappings to preserve Controls.Page legacy behaviors
		PageHandler.Mapper.ReplaceMapping(PlatformConfiguration.iOSSpecific.Page.SafeAreaInsetsProperty.PropertyName, MapUpdateSafeAreaInsets)
	}
```

Reply:
Updated! @mattleibow @PureWeen